src/components/calculator.js

```diff
@@ -1,18 +1,39 @@
 import React, { useState } from 'react';
 import './calculator.css';
+// import calculate from '../logic/calculate';
```

Comment:
- Please uncomment this line and instead remove the next import line. This is because `operate.js` is already been imported in the `calculate.js` file.